modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc

```diff
@@ -127,12 +128,36 @@ download/example.sh:::
 # You can customitze the main html file and re-run repo-admin.py later.
 # For a server without https support use --insecure --url http://...
 #
+set -euxo pipefail
+
 cd "$(dirname "${0}")"
-set -eux
-repo=http://download.example.com/repo
-python3 bin/repo-admin.py --url $repo -d 'download' -p '.*-client' -i 'index.html' -f ..
+
+opt=()
+
+# if a .repo-admin-prefix file is found in a parent directory, --url can be omitted
+opt+=("--url" "http://download.example.com/repo")
+
+python3 bin/repo-admin.py  "${opt[@]}" -d 'download' -p '.*-client' -i 'index.html' -f ..
 ----
 
-Replace `\http://download.example.com/repo` in the above example with the base URL of your repository (ending in `.../mycloud-2.10.0.6752-linux/` here) and save the file.
+The url after unpacking is an internal minio url, or a non working example like `\http://download.example.com/repo`.
```

Comment:
We try not to include any internally available URLs anymore. It also doesn't really matter whether there's a broken URL in place or none at all.